mongo/mongolog/default_logger.go

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) MongoDB, Inc. 2017-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+package mongolog
+
+import (
+	"fmt"
+	"io"
+)
+
+// defaultLogger is the default logger for the mongo package. By default, it sends logs to stdErr,
+// but can be configured to log to files with SetOutputFile Mongologger option.
+// Logs are of the format {level:<level>,msg:<message>,<fieldName>:<fieldValue>,...}
```

Comment:
I think the format will depend on the type of log message. IIUC the default string output should be unstructured and follow the formats in the spec:

> "{{commandName}} command succeeded with request ID {{requestId}}. Server reply: {reply}}. Operation ID: {{operationId}}, driver connection ID: {{driverConnectionId}}, server connection ID: {{serverConnectionId}}, explicit session: {{explicitSession}}, duration: {{duration}}"

I'm not sure off-hand what the best way would be to convey that format to the default logger.